src/components/layers/toolbar/LayerToolbarMoreMenu.js

```diff
@@ -1,190 +1,119 @@
-                            </ListItemIcon>
-                            <ListItemText primary={i18n.t('Data table')} />
-                        </MenuItem>
-                    )}
-                    {openAs && (
-                        <MenuItem
-                            onClick={this.handleOpenAsChartBtnClick}
-                            className={classes.menuItem}
-                        >
-                            <ListItemIcon>
-                                <ChartIcon />
-                            </ListItemIcon>
-                            <ListItemText primary={i18n.t('Open as chart')} />
-                        </MenuItem>
-                    )}
-                    {downloadData && (
-                        <MenuItem
-                            onClick={this.handleDownloadBtnClick}
-                            className={classes.menuItem}
-                        >
-                            <ListItemIcon>
-                                <SaveIcon />
-                            </ListItemIcon>
-                            <ListItemText primary={i18n.t('Download data')} />
-                        </MenuItem>
-                    )}
-                    {showDivider && (
-                        <Divider className={classes.divider} light />
-                    )}
-                    {onEdit && (
-                        <MenuItem
-                            onClick={this.handleEditBtnClick}
-                            className={classes.menuItem}
-                        >
-                            <ListItemIcon>
-                                <EditIcon />
-                            </ListItemIcon>
-                            <ListItemText primary={i18n.t('Edit layer')} />
-                        </MenuItem>
-                    )}
-                    {onRemove && (
-                        <MenuItem
-                            onClick={this.handleRemoveBtnClick}
-                            className={classes.menuItem}
-                        >
-                            <ListItemIcon>
-                                <DeleteIcon />
-                            </ListItemIcon>
-                            <ListItemText primary={i18n.t('Remove layer')} />
-                        </MenuItem>
-                    )}
-                </Menu>
-            </Fragment>
-        );
-    }
-}
+                    <MoreIcon />
+                </div>
+            </Tooltip>
+            {isOpen && (
+                <Popover
+                    reference={anchorRef}
+                    arrow={false}
+                    placement="right"
+                    onClickOutside={() => setIsOpen(false)}
+                >
+                    <div className={styles.menu}>
+                        <Menu dense>
+                            {toggleDataTable && (
+                                <MenuItem
+                                    label={i18n.t('Data table')}
```

Comment:
I'd adjust this label to "Show data table" so that all menu items are consistent with "action + thing".

Reply:
Fixed.